docs/conf/prague/2022/news/get-ready.rst

```diff
@@ -0,0 +1,71 @@
+
+Check out the `Schedule page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/schedule/>`_ for exact time and locations.
+
+All our events and activities
+-----------------------------
+
+There are a large number of events at the conference beyond the main track of speakers.
+We hope that you'll get involved with some of our other events,
+and get to know more folks that are hanging out at the conference!
+
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/lightning-talks/>`__ each day after lunch. Registration is handled on the day of the conference. These are 5-minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
+* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. These are also planned day-of, and allow attendees to schedule a session to talk about any topic they wish. It's a great way to find others who share your specific interests. As a session lead you guide the session, but you definitely do not need to have a full plan.
+* **Job Fair**. On Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/job-fair>`_ where you can find your next documentarian-friendly job. Many of our sponsors will have a table, where you can learn more about working with them.
+* **Social event**. On Monday evening we'll be moving to a spatial chat platform for our social event. This will be a more familiar social space where you can move around and join groups of people having conversations. We've used similar platforms at other events, and found that they really help people get to know each other in a more natural way.
+* **Conference Help Desk**. If you have any questions about navigating the virtual event, we'll be running the Write the Docs Helpdesk booth in the **Expo** area, where you can chat with one of the organizers and the Welcome Wagon team.
+
+The `Schedule page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/schedule/>`_ has the latest and most complete information about all our events. This is where we will also update any changes or cancellations if they should occur.
+
+Thanks so much to all our sponsors
+----------------------------------
+
+We are so grateful to have our sponsors help in bringing these events to life every year.
+You'll be able to chat with most of our sponsors at the Job Fair, and a few will have booths for conversations the entire conference.
+
+Thanks so much to the following companies for supporting the Portland conference this year:
+
+.. datatemplate::
+   :source: /_data/{{shortcode}}-{{year}}-config.yaml
+   :template: {{year}}/sponsors-simplelist.rst
+
```

Comment:
Sorry folks, haven't had a chance to look at this, #1798 contains a speaker name change, so would be better to merge that before this one.